purchase_report_custom_hls/report/purchase_order_report_templates.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document_inherit_purchase_order"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//div[@t-if='o.date_order']" position="before">
+            <div t-if="o.payment_term_id">
+                <strong>Payment Terms:</strong>
+                <br />
+                <span t-field="o.payment_term_id"/>
+            </div>
+        </xpath>
+        <xpath expr="//div[@class='page']" position="after">
+            <div t-if="o.company_id.rep_title">
+                <tr>
+                    <hr
+                        width="100%"
+                        style="border: none; border-bottom: 2px solid black;"
+                    />
+                    <td width="100%">
+                        <div style="text-align: right; padding-top: 15px padding-right: 10px">
+                            <img
+                                t-if="o.company_id.rep_signature"
+                                t-att-src="image_data_uri(o.company_id.rep_signature)"
+                                style="max-height: 40px;"
+                                alt="Signature"
+                            />
```

Comment:
Why the signature below the border line? It should be above.

Reply:
Sorry, I edited it.
Layout wasn't same as expected, so added parent class and edit styles.